randovania/games/samus_returns/logic_database/Area 4 - West.txt

```diff
@@ -284,14 +287,15 @@ Extra - asset_id: collision_camera_006
       Morph Ball and Spider Ball and After Area 4 (West) - Chozo Seals Grapple Block Right
   > Right of Left Elevator
       Any of the following:
-          Space Jump or Can Spiderspark
+          Space Jump
           All of the following:
               Morph Ball
               Any of the following:
                   After Area 4 (West) - Chozo Seals Grapple Block Pull Left
                   All of the following:
                       Spider Ball
                       Bomb or Spring Ball or Mid-Air Morph (Beginner)
+          Spiderspark (Intermediate) and Can Spiderspark
   > Event - Grapple Block Left
       Grapple Beam
 
```

Comment:
I can't add a line comment for it, but also add movement beginner to the Space Jump paths connecting `Right of Left Elevator` -> `Left Chozo Seal` and `Chozo Seal Right` -> `Chozo Seal Left`

Reply:
Added